components/brave_wallet_ui/utils/reduce-account-name.ts

```diff
@@ -0,0 +1,13 @@
+export const reduceAccountDisplayName = (name: string, reduceBy: number) => {
```

Comment:
Are really doing a `reduceBy`? Looks like `maxLength` would be more appropriate. Also maybe we should take into account the `..` and discount it in the `maxLength`, so you may have to adjust the values used in different places accordingly.

Reply:
Done!